content/blog/2021/11/2021-11-12-interview-duchess-hacktoberfest.adoc

```diff
@@ -0,0 +1,157 @@
+The fourth reviewer did find an issue.
+Who can tell me code review isn’t useful now?
+
+From a personal perspective, I remember two things.
+The first one: Open Source communities are not elitist communities.
+Every contribution matters, even small ones.
+The people I met were positive and benevolent.
+The second one could be illustrated by a quote from Grace Hopper: “If it's a good idea, go ahead and do it. It's much easier to apologize than it is to get permission.”.
+
+=== Do you have anything more to say?
+After Jenkins, I also submitted Pull Requests to others Open Source projects.
+The problem: with such a diversity of Open Source projects, how to organize your time?
+
+=== Anne-Laure Gaillard contributions during hacktoberfest
+* https://github.com/jenkinsci/jenkins/pull/5779
+* https://github.com/jenkinsci/jenkins/pull/5785
+* https://github.com/jenkinsci/jenkins/pull/5804
+* https://github.com/jenkinsci/jenkins/pull/5806
+* https://github.com/jenkinsci/jenkins/pull/5807
+* https://github.com/jenkinsci/jenkins/pull/5808
+* https://github.com/jenkinsci/jenkins/pull/5822
+* https://github.com/jenkinsci/jenkins/pull/5823
+* https://github.com/mattermost/quality-assurance/pull/25
+* https://github.com/mattermost/quality-assurance/pull/23
+* https://github.com/sarequl/Netflix-Helper/pull/6
+
+== Bertha Torres Interview
+
+image:/images/post-images/2021-11-duchess-hacktoberfest/tb.jpg[Bertha Torres picture, height=200]
+
+* link:https://twitter.com/tatoberres[Twitter: @tatoberres]
+* link:https://www.linkedin.com/in/bertha-torres-62b672209/[LinkedIn]
+
+=== Hello Bertha, can you introduce yourself?
+Coming from a literature training and as a language teacher for more than 15 years, it’s while moving to a high mountain area that a need to explore other skills set in.
+
+Thus, a self-taught training followed by a validated file for financing diploma training followed one another and confirmed to me that life is too short to waste it on what does not make sense to us. 
+
+=== What did you think about Open Source, and maybe Jenkins, before starting the hackathon?
+Participating in Open Source appeared to me in that exploratory process like a learning opportunity… 
+which I thought was inaccessible of course! I didn’t see myself writing code.
+
+But thanks to the Duchess France community, I learned how to contribute at my level and to understand the contribution process.
+It’ll allow me as things progress to contribute in more domains.
+
+=== Can you tell us how the woman version of Jenkins was born?
+
+Well… In the presentation video for Hacktoberfest, the artwork was mentioned (https://www.jenkins.io/artwork/) and Angélique also mentioned it during the first meeting… 
+And one must admit that looking at it, you can see a lot of different things but women.
+
+So I imagined a governess, very confident, efficient, a bit clever, scrupulously combed, nicely dressed, who might represent the personality of what Jenkins is made of.
```

Comment:
A bit clever -> Can we use instead "very clever" ?

Reply:
```suggestion
So I imagined a governess, very confident, efficient, very clever, scrupulously combed, nicely dressed, who might represent the personality of what Jenkins is made of.
```
yes sure !